content/components/overlay.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: Overlay
+status: Alpha
+---
+
+import {Grid, Flex, Box, Button, ButtonOutline, Heading, Label, LabelGroup, Link} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    An overlay is a flexible floating surface, used to display transient content such as menus, selection options, dialogs, and more. Overlays use shadows to express elevation, and may include a backdrop that dims the rest of the screen for focus. 
+</Box>
+
+<LabelGroup display="block" mb={4}>
+    <Label as="a" href="https://primer.style/react/Overlay" variant="xl" color="text.success" outline style="text-decoration: none; line-height: 20px;">
+        <img width="20" height="20" alt=" " src="https://user-images.githubusercontent.com/293280/123878374-ce9d4d00-d8f3-11eb-8adf-1a160292ff53.png" style="vertical-align: middle; margin-right: 4px;" />
+        React
+    </Label>
+    <Label as="a" href="https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?node-id=1927%3A0" variant="xl" outline style="text-decoration: none; line-height: 20px;">
+        <img width="20" height="20" alt=" " src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png" style="vertical-align: middle; margin-right: 4px;" />
+        Figma
+    </Label>
+</LabelGroup>
+
+<img src="https://user-images.githubusercontent.com/293280/125706076-b16bb3d4-800f-4310-a082-f576e215e6c8.png" alt="Overlay examples" />
+
+## Overview
+
+Overlays can streamline the interface by allowing extra options to be displayed as needed. Overlays present a new context to the user, and can facilitate the completion of individual tasks. They are also useful for data-manipulation, providing a dedicated surface for selecting or editing options.
+
+**Keep overlays simple**. Overlays are meant to be interacted with quickly, so they're better suited for clear experiences. Avoid creating a whole page inside an overlay. Prefer single-column layouts that convey a clear goal the moment it's opened.
+
+**Overlays can be fully interacted with a keyboard**. Keyboard navigation should be limited to the elements of the active overlay. If the overlay contains a list, let users navigate using keyboard arrows. Pressing Escape should dismiss the overlay. When an overlay is dismissed, focus generally returns to the element that opened it.
+
+**Overlay contents should work well on any device**. Overlays may adapt to bottom sheets or full-screen on mobile devices. Make sure the overlay contents work with all supported sizes and input methods.
+
+
+## Anatomy
+
+Overlays may include a header to provide context, and footer to host actions. The content area supports vertical scrolling when showing lots of items and longer forms. For rendering interactive lists, use [Action lists](/components/action-list).
```

Comment:
```suggestion
Overlays may include a header to provide context, and footer to host actions. The content area supports vertical scrolling when showing lots of items and longer forms. For rendering interactive lists, use an [action list](/components/action-list).
```

Reply:
The copy here has been cleaned up and no longer mentions action lists.